rest/importtest/import_test.go

```diff
@@ -1916,6 +1917,141 @@ func TestImportRevisionCopyUnavailable(t *testing.T) {
 	assert.Equal(t, 404, response.Code)
 }
 
+func TestImportComputeStatOnDemandGet(t *testing.T) {
+	base.SkipImportTestsIfNotEnabled(t)
```

Comment:
This I don't think requires the import feed, so I don't know that this is correct, see also OnDemandPut, can you test this with removing this and `SG_TEST_USE_XATTRS=false`?

Reply:
Its needed -> https://github.com/couchbase/sync_gateway/blob/8bc18bd35833deea85b6f21586f0b8de7059a0f4/db/database.go#L1911

Reply:
Test will panic without it. SharedBucketImportStats don't exist without XATTRS. We would need to move the stats to collection stats or database stats or something if this is needed.

Reply:
This is weird, but there are just no stats with xattrs, I'm inclined to push deprecation of inline data in sync gateway. 

Thanks for explanation, that was not intuitive to me.